ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -43,6 +43,7 @@ import {SuspendInputs} from './suspend-inputs';
 import {WorkflowResourcePanel} from './workflow-resource-panel';
 
 import './workflow-details.scss';
+import {RetryWorkflowNode} from '../retry-workflow-node-panel';
```

Comment:
I've said this [before](https://github.com/argoproj/argo-workflows/pull/13160#discussion_r1633495520): please follow the existing import style where these are grouped together and sorted.

Reply:
I had to manually fix this in https://github.com/argoproj/argo-workflows/pull/13813 as it is unsortable by tooling since it was placed after a side effect import